src/modules/blocks/event-datetime/subblocks/content/template.js

```diff
@@ -42,219 +43,219 @@ FORMATS.date = settings() && settings().dateWithYearFormat
 	? settings().dateWithYearFormat
 	: __( 'F j', 'the-events-calendar' );
 
-class EventDateTimeContent extends Component {
-	static propTypes = {
-		allDay: PropTypes.bool,
-		cost: PropTypes.string,
-		currencyPosition: PropTypes.oneOf( [ 'prefix', 'suffix', '' ] ),
-		currencySymbol: PropTypes.string,
-		end: PropTypes.string,
-		isEditable: PropTypes.bool,
-		multiDay: PropTypes.bool,
-		onDateTimeLabelClick: PropTypes.func,
-		sameStartEnd: PropTypes.bool,
-		separatorDate: PropTypes.string,
-		separatorTime: PropTypes.string,
-		setCost: PropTypes.func,
-		setTimeZoneLabel: PropTypes.func,
-		showDateInput: PropTypes.bool,
-		showTimeZone: PropTypes.bool,
-		start: PropTypes.string,
-		timeZone: PropTypes.string,
-		timeZoneLabel: PropTypes.string,
-	};
-
-	renderPrice = () => {
-		const { cost, currencyPosition, currencySymbol, setCost } = this.props;
+/**
+ * Renders a separator based on the type called
+ *
+ * @param {string} type - The type of separator
+ *
+ * @returns {ReactDOM} A React Dom Element null if none.
+ */
+const renderSeparator = ( props, type, className ) => {
+	const { separatorDate, separatorTime } = props;
 
-		// Bail when not classic
-		if ( ! editor() || ! editor().isClassic ) {
+	switch ( type ) {
+		case 'date-time':
+			return (
+				<span className={ classNames( 'tribe-editor__separator', className ) }>
+					{ ' '.concat( separatorDate, ' ' ) }
```

Comment:
If we're adding a space before and after, I think we could just do
```suggestion
					{` ${separatorDate} ` }
```